ionc/ion_reader_text.c

```diff
@@ -836,7 +836,7 @@ iERR _ion_reader_text_step_out(ION_READER *preader)
             sub_type = IST_LIST;
             break;
         default: {
-            char* error_message[ION_ERROR_MESSAGE_MAX_LENGTH];
+            char error_message[ION_ERROR_MESSAGE_MAX_LENGTH];
```

Comment:
This is just a case of declaring char*[] (which is essentially char**) when we meant char[].